packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -311,10 +311,14 @@ export default class QuanticSearchBoxInput extends LightningElement {
 
   clearInput() {
     this.sendInputValueChangeEvent('');
+    this.setDisplayedInputValue('');
+    this.onBlur();
     this.input.focus();
```

Comment:
it's quite confusing/weird to have a line that focuses the input right after blurring it. so I think this could be better.

Reply:
Instead that's what we can do.
```suggestion
    this.input.removeAttribute('aria-activedescendant');
    this.collapseTextArea();
    this.input.focus();
```

This way we are not blurring the input but we are doing only what needs to be done.